cli/deployment/config.go

```diff
@@ -406,7 +406,7 @@ func newConfig() *codersdk.DeploymentConfig {
 			Usage:   "How frequently agent stats are recorded",
 			Flag:    "agent-stats-refresh-interval",
 			Hidden:  true,
-			Default: 10 * time.Minute,
+			Default: 30 * time.Second,
```

Comment:
This is a pretty big change, I think it's OK but increases spamminess somewhat.

Reply:
It definitely is, but I did some napkin math and I think it _should_ be alright.

Even if a user has hundreds of workspaces, a few hundred more writes/minute shouldn't be a big deal. I suppose it might spam the logs, which I'll check and resolve before merge.

Reply:
I'd hate for all of coderd to be spammed with stat logs in a large deployment ;p